git_tools/create_version_branch.py

```diff
@@ -163,8 +163,4 @@ def _create_version_branch(arg_strings=None):# pylint: disable=too-many-branches
         raise exc
 
 if __name__ == '__main__':
-    try:
-        _create_version_branch()
-    except Exception as exc:# pylint: disable=broad-except
-        print("Error: ", exc)
-        sys.exit(1)
+    _create_version_branch()
```

Comment:
Catching the exception just hides stack trace info that would be useful in the workflow logs.